server/views/pages/prisons/addSessionTemplate.njk

```diff
@@ -0,0 +1,236 @@
+          id: "templateName",
+          name: "templateName",
+          autocomplete: "off",
+          errorMessage: errors | findError('templateName')
+        }) }}
+
+        {{ govukSelect({
+          id: "dayOfWeek",
+          name: "dayOfWeek",
+          label: {
+            text: "Day of the week",
+            classes: "govuk-label--m"
+
+          },
+          items: [
+            {
+              value: "choose",
+              text: "Choose day of the week",
+              selected: true
+            },
+            {
+              value: "MONDAY",
+              text: "Monday"
+            },
+            {
+              value: "TUESDAY",
+              text: "Tuesday"
+            },
+            {
+              value: "WEDNESDAY",
+              text: "Wednesday"
+            },
+            {
+              value: "THURSDAY",
+              text: "Thursday"
+            },
+            {
+              value: "FRIDAY",
+              text: "Friday"
+            },
+            {
+              value: "SATURDAY",
+              text: "Saturday"
+            },
+            {
+              value: "SUNDAY",
+              text: "Sunday"
+            }
+          ]
+
+        }) }}
+
+        {{ govukInput({
+          label: {
+              text: "Visit start time",
+              classes: "govuk-label--m"
+          },
+          hint: {
+              text: "What is the visit start time? (HH:MM)"
+          },
+          classes: "govuk-!-width-one-third",
+          id: "startTime",
+          name: "startTime",
+          autocomplete: "off",
+          errorMessage: errors | findError('startTime')
+        }) }}
+
+        {{ govukInput({
+          label: {
+              text: "Visit end time",
+              classes: "govuk-label--m"
+          },
+          hint: {
+              text: "What is the visit end time? (HH:MM)"
+          },
+          classes: "govuk-!-width-one-third",
+          id: "endTime",
+          name: "endTime",
+          autocomplete: "off",
+          errorMessage: errors | findError('endTime')
+        }) }}
+
+        {{ govukRadios({
+          name: "biWeekly",
+          fieldset: {
+            legend: {
+              text: "Bi-weekly?",
+              classes: "govuk-fieldset__legend--m"
+            }
+          },
+          items: [
+            {
+              value: "true",
+              text: "True"
+            },
+            {
+              value: "false",
+              text: "False"
+            }
+          ]
+        }) }}
+
+        {{ govukDateInput({
+          id: "validFromDate",
+          namePrefix: "validFromDate",
+          fieldset: {
+            legend: {
+              text: "When is the session template valid from?",
+              classes: "govuk-fieldset__legend--m"
+            }
+          },
+          hint: {
+            text: "For example, 27 3 2023"
+          }
+        }) }}
+
+        {{ govukDateInput({
```

Comment:
Forgot earlier when were discussing that a template doesn't have to have an end date. So, need to cater for this. Perhaps title could be "Template end date" and below it is a checkbox that is labelled "Add an end date". This then reveals the text input when clicked. Like BaPV does when you add 'Other' support details. 

See GDS page here about "Conditional reveal" - https://design-system.service.gov.uk/components/checkboxes/